src/Opus.hpp

```diff
@@ -38,15 +38,15 @@ template< typename T > class OpusBase {
 	std::unique_ptr< T, Destructor > m_ctx;
 };
 
-class Opus::Decoder::P : public OpusBase< ::OpusDecoder > {
+class Opus::Decoder::P : public OpusBase<::OpusDecoder > {
```

Comment:
A bug in `clang-format`?

Reply:
Yeah, that's a quirk of clang-format v10

Reply:
I wonder what version I had when all the extra spaces were added...

Reply:
I believe this issue is fixed from version 11 onwards, but I am not quite sure.
However, I would argue that we should stay consistent with our setup in main Mumble in that regard.

Reply:
Yes, absolutely.